tidb-server/main.go

```diff
@@ -220,6 +221,23 @@ func main() {
 	syncLog()
 }
 
+func planCacheInfo(svr *server.Server) {
+	defer func() {
+		if r := recover(); r != nil {
+			logutil.BgLogger().Warn("[Plan-Cache-Patch] server plan cache info panic", zap.Any("panic", r))
+		}
+	}()
+
+	for {
+		<-time.After(time.Second * 30)
```

Comment:
We will require lock in `Server.PlanCacheInfo`, it may influence the performance(I guess?). Maybe every 30s to execute once is more frequently?

Reply:
Update it to 2mins, PTAL.